book/4 - MultiWorlds/3 - Linking Worlds with a Plugin.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The easiest way to link worlds together is using a dedicated plugin, such as <a href="http://forum.mc-server.org/showthread.php?tid=2157">Portal V2</a>. More plugins can be found at the <a href="http://forum.mc-server.org/forumdisplay.php?fid=2">respository</a>.
```

Comment:
The plugin is much more adaptable - you can configure it so that some portals go to one world and some to another. That isn't possible with world linking, and many servers will want that configurability.